radixdlt-java/src/test/java/com/radixdlt/client/core/serialization/DsonTest.java

```diff
@@ -14,131 +14,10 @@
kRlZmluaXRpb25SZWZlcmVuY2VYQAYvSkgxUDhmM3puYnlyRGo4RjRSV3BpeDdoUmtneHFIamRXMmZObktwUjN2NnVmWG5rbm9yL3Rva2Vucy9YUkRndmVyc2lvbhhk/2pzZXJpYWxpemVyc3JhZGl4LnNwdW5fcGFydGljbGVkc3BpbgFndmVyc2lvbhhk/2pzZXJpYWxpemVydHJhZGl4LnBhcnRpY2xlX2dyb3VwZ3ZlcnNpb24YZP+/aXBhcnRpY2xlc4O/aHBhcnRpY2xlv2ZhbW91bnRYIQX//////////////////////////////////////////2tncmFudWxhcml0eVghBQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABZW5vbmNlGwAAjcu1625Ma3Blcm1pc3Npb25zv2RidXJuZG5vbmVkbWludHN0b2tlbl9jcmVhdGlvbl9vbmx5/2pzZXJpYWxpemVyeCJyYWRpeC5wYXJ0aWNsZXMudW5hbGxvY2F0ZWRfdG9rZW5zeBh0b2tlbkRlZmluaXRpb25SZWZlcmVuY2VYQAYvSkgxUDhmM3puYnlyRGo4RjRSV3BpeDdoUmtneHFIamRXMmZObktwUjN2NnVmWG5rbm9yL3Rva2Vucy9YUkRndmVyc2lvbhhk/2pzZXJpYWxpemVyc3JhZGl4LnNwdW5fcGFydGljbGVkc3BpbiBndmVyc2lvbhhk/79ocGFydGljbGW/ZmFtb3VudFghBf///////////////////////////MTRw2Avf8MX////a2dyYW51bGFyaXR5WCEFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFlbm9uY2UbAACNy7XrnqhrcGVybWlzc2lvbnO/ZGJ1cm5kbm9uZWRtaW50c3Rva2VuX2NyZWF0aW9uX29ubHn/anNlcmlhbGl6ZXJ4InJhZGl4LnBhcnRpY2xlcy51bmFsbG9jYXRlZF90b2tlbnN4GHRva2VuRGVmaW5pdGlvblJlZmVyZW5jZVhABi9KSDFQOGYzem5ieXJEajhGNFJXcGl4N2hSa2d4cUhqZFcyZk5uS3BSM3Y2dWZYbmtub3IvdG9rZW5zL1hSRGd2ZXJzaW9uGGT/anNlcmlhbGl6ZXJzcmFkaXguc3B1bl9wYXJ0aWNsZWRzcGluAWd2ZXJzaW9uGGT/v2hwYXJ0aWNsZb9nYWRkcmVzc1gnBAIDeFqcJZ/emZHkT6L7C1ZZ8qV4GsM5B24tv+9wUo5K32iIecG5ZmFtb3VudFghBQAAAAAAAAAAAAAAAAAAAAAAAAAAAzsuPJ/QgDzoAAAAa2dyYW51bGFyaXR5WCEFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFlbm9uY2UbAACNy7Xrl6BrcGVybWlzc2lvbnO/ZGJ1cm5kbm9uZWRtaW50c3Rva2VuX2NyZWF0aW9uX29ubHn/ZnBsYW5jaxoBeoBAanNlcmlhbGl6ZXJ4JHJhZGl4LnBhcnRpY2xlcy50cmFuc2ZlcnJhYmxlX3Rva2Vuc3gYdG9rZW5EZWZpbml0aW9uUmVmZXJlbmNlWEAGL0pIMVA4ZjN6bmJ5ckRqOEY0UldwaXg3aFJrZ3hxSGpkVzJmTm5LcFIzdjZ1Zlhua25vci90b2tlbnMvWFJEZ3ZlcnNpb24YZP9qc2VyaWFsaXplcnNyYWRpeC5zcHVuX3BhcnRpY2xlZHNwaW4BZ3ZlcnNpb24YZP9qc2VyaWFsaXplcnRyYWRpeC5wYXJ0aWNsZV9ncm91cGd2ZXJzaW9uGGT/anNlcmlhbGl6ZXJqcmFkaXguYXRvbWpzaWduYXR1cmVzv3ggNTZhYmFiMzg3MDU4NWYwNGQwMTVkNTVhZGY2MDBiYze/YXJYIQF/LcbiXLx5V5N9s3vIGmvvuTJMcqzOCFuMG52k2evOhGFzWCEB+3a22OJO5aOZioTllTQmjWHXp0Kemu1c5EsZ4ojtnuFqc2VyaWFsaXplcnZjcnlwdG8uZWNkc2Ffc2lnbmF0dXJlZ3ZlcnNpb24YZP//bXRlbXBvcmFsUHJvb2a/Zm9iamVjdFghA7/4tBnGJq8xG5HR1P+yUN/+WHi1dN4Jn8k1fUgBiTtLanNlcmlhbGl6ZXJ0dGVtcG8udGVtcG9yYWxfcHJvb2ZndmVyc2lvbhhkaHZlcnRpY2Vzgb9lY2xvY2sAamNvbW1pdG1lbnRYIQMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGVvd25lclgiAQMFw9atcKr8nO0o+qPEBenUtjD6YXuaehQ/JmpFuZJFMWhwcmV2aW91c1ECAAAAAAAAAAAAAAAAAAAAAGZyY2xvY2sbAAABWocqmABqc2VyaWFsaXplcnV0ZW1wby50ZW1wb3JhbF92ZXJ0ZXhpc2lnbmF0dXJlv2FyWCEB2xz/KUZqI2PHDeFjQ+wW4+cd8q8+rZ7qG5LVkAYn65Jhc1ghAbisc1QOC7RM8L7/aeBle4vcQ5dnlZHj2+9JCQUI28bSanNlcmlhbGl6ZXJ2Y3J5cHRvLmVjZHNhX3NpZ25hdHVyZWd2ZXJzaW9uGGT/anRpbWVzdGFtcHO/Z2RlZmF1bHQbAAABWocqmAD/Z3ZlcnNpb24YZP//Z3ZlcnNpb24YZP9lbWFnaWMaA82AAmRuYW1lbFJhZGl4IERldm5ldGZwbGFuY2sZ6mBkcG9ydBl1MGpzZXJpYWxpemVybnJhZGl4LnVuaXZlcnNla3NpZ25hdHVyZS5yWCEB8+ZGK30jwrDYRR3INLxXftu7HwL8jPHEmyJmEhM8gy9rc2lnbmF0dXJlLnNYIQHkWi4nrVupHGA0g5Z6J/736Su6Gwzjh/ZNZy/xH7KvYWl0aW1lc3RhbXAbAAABWocqmABkdHlwZQJndmVyc2lvbhhk/w=="
```

Comment:
Surely checkstyle complains about this?

Reply:
Would be better to change this to a round-trip test, with some additional field validation IMO.

Reply:
It does complain, haven't gotten to fixing this yet. Will do.